internal/uvm/constants.go

```diff
@@ -29,9 +31,43 @@ const (
 	WCOWGlobalMountPrefix = "C:\\mounts\\m%d"
 	// RootfsPath is part of the container's rootfs path
 	RootfsPath = "rootfs"
+
+	// Maximum number of SCSI controllers allowed
+	MaxSCSIControllers = 4
```

Comment:
Can we remove this and just use `len(SCSI_CONTROLLER_GUIDS)` (or `len(scsiControllerGUIDs)`) instead?

Reply:
It won't be a constant then but that should be okay I guess.